python/peacock/base/MediaControlWidgetBase.py

```diff
@@ -223,7 +223,7 @@ def __addButton(self, name, tooltip, icon):
     def __addEditBox(self, name, label, tooltip):
         edit = QtWidgets.QLineEdit()
         edit.setToolTip(tooltip)
-        edit.setFixedWidth(152)
+        edit.setSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
```

Comment:
These are the time and timestep edit boxes, they should get as much space as available because there could be large numbers in them.